UnitTests/GitCommandsTests/GitModuleTest.cs

```diff
@@ -64,22 +64,23 @@ public void ParseGitBlame()
             Assert.AreEqual("", result.Lines.Last().Text);
         }
 
-        [Test]
-        public void UnescapeOctalCodePoints_handles_octal_codes()
+        [TestCase(null, "")]
+        [TestCase("", "")]
+        [TestCase(" ", " ")]
+        [TestCase("Hello, World!", "Hello, World!")]
+        [TestCase(@"\353\221\220\353\213\244.txt", "두다.txt")] // escaped octal code points (Korean Hangul in this case)
+        [TestCase(@"Invalid byte \777.txt", @"Invalid byte \777.txt")] // 777 is an invalid byte, which is omitted from the output
```

Comment:
:bulb: Would be good to add one showing a good escape sequence along with a bad one.

Reply:
Done.